service-commands/scripts/setup-instance.sh

```diff
@@ -138,25 +99,26 @@ case "$service" in
 			"[[ ! -d ~/audius-protocol ]]" \
 			"&& set -x" \
 			"&& git clone https://github.com/AudiusProject/audius-protocol.git" \
-			"&& PROTOCOL_DIR=/home/ubuntu/audius-protocol bash ~/audius-protocol/service-commands/scripts/set-git-refs.sh $audius_protocol_git_ref $audius_client_git_ref" \
-			"&& yes | bash audius-protocol/service-commands/scripts/provision-dev-env.sh" || true
-        reboot_instance $provider $name
+			"&& PROTOCOL_DIR=audius-protocol bash ~/audius-protocol/service-commands/scripts/set-git-refs.sh $audius_protocol_git_ref $audius_client_git_ref" \
+			"&& yes | bash audius-protocol/service-commands/scripts/provision-dev-env.sh"
+		reboot_instance $provider $name
 
 		wait_for_instance $provider $user $name
 
 		# perform once more for audius-client
 		execute_with_ssh $provider $user $name \
-			"PROTOCOL_DIR=/home/ubuntu/audius-protocol bash ~/audius-protocol/service-commands/scripts/set-git-refs.sh $audius_protocol_git_ref $audius_client_git_ref"
+			"bash ~/audius-protocol/service-commands/scripts/set-git-refs.sh $audius_protocol_git_ref $audius_client_git_ref"
 
-		execute_with_ssh $provider $user $name "nohup npm run start:dev:cloud > ~/audius-client.out 2>&1 &"
-		execute_with_ssh $provider $user $name "source ~/.nvm/nvm.sh; source ~/.profile; A up || (A down && A up)"
-
-		copy_file_to_remote $provider $user $name "~/.gitconfig" "~/.gitconfig"
+		# start up frontend and backend
+		execute_with_ssh $provider $user $name "set -x; nohup npm run start:dev:cloud > ~/audius-client.out 2>&1 &"
+		execute_with_ssh $provider $user $name "set -x; source ~/.nvm/nvm.sh; source ~/.profile; A up"
 
+		# configure local files: /etc/hosts, /etc/ssh/ssh_config.d/60-audius.conf
 		configure_etc_hosts
-
 		set_ssh_serveralive
 
+		# upload personal files: ~/.gitconfig, ~/.zshrc, ~/.zshenv, ~/.p10k.zsh
+		copy_file_to_remote $provider $user $name "~/.gitconfig" "~/.gitconfig"
```

Comment:
This is extremely dangerous and should not be done -> user's may have tokens and keys on their local gitconfigs that should be regenerated on a machine.

Reply:
O_O why would you set it up like that??

But fair enough. Thanks for letting me know this is a possibility! 

I'll do logic around a `~/.gitconfig.remote-dev` check and only upload that file if it exists.